packages/camera/camera_avfoundation/example/ios/RunnerTests/Mocks/MockCaptureVideoDataOutput.swift

```diff
@@ -0,0 +1,23 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Mock implementation of `FLTCaptureVideoDataOutput` protocol which allows injecting a custom
+/// implementation.
+class MockCaptureVideoDataOutput: NSObject, FLTCaptureVideoDataOutput {
+
+  var avOutput: AVCaptureVideoDataOutput = AVCaptureVideoDataOutput()
+  var alwaysDiscardsLateVideoFrames: Bool = false
```

Comment:
nit: for this and above, can infer type

Reply:
For the array, do you prefer the current
`var videoSettings: [String: Any] = [:]`
over
`var videoSettings = [String: Any]()`?

Reply:
i think both are fine. I don't see it in the style guide.